cluster/saltbase/salt/docker/init.sls

```diff
@@ -47,7 +47,7 @@ docker:
       - pkg: docker-io
 
 {% endif %}
-{% elif grains.cloud is defined and grains.cloud == 'vsphere' and grains.os == 'Debian' and grains.osrelease_info[0] >=8 %}
+{% elif grains.cloud is defined and (grains.cloud == 'vsphere' or grains.cloud == 'photon-controller') and grains.os == 'Debian' and grains.osrelease_info[0] >=8 %}
```

Comment:
i prefer `grains.cloud in ['vsphere', 'photon-controller'] its a bit less verbose